README.md

```diff
@@ -1 +1,66 @@
 # wizard-additional-playing-cards
+
+## Tornado | 14 | blau
+
+- Nach dem Stich wird in die andere Richtung gespielt
+- Bis zum Ende der Runde
+
+## Katapult | 14 | rot
+
+- Alle ziehen eine Karte
+
+## Frosch | 14 | grün
+
+- Alle müssen eine Karte auf den Stapel legen
+
+## Affe | 14 | gelb
+
+- Beim ausspielen wird die Farbkarte umgedreht
+
+## Nagel+Hammer | N | weiß
+
+- Der aktuelle Stich bleibt liegen und der nächste Sich wird darauf weiter gespielt
+- Der Gewinner bekommt 2 Stiche
+
+## Wage | 4/11 | bunt
+
+- Die Niedrigste Karte gewinnt
+- Bei zwei Narren gewinnt der 1. Narre
+
+## Rauchbombe | 4.20 | weiß
+
+- Farben sind egal
+- Bei zwei gleich großen Karten gewinnt die Zuerst gespielte
+
+## Marionette | <> | weiß
+
+- Wird diese Karte gespielt, wird eine Karte vom Stapel stattdessen gespielt
+
+## Schütze | 11.1 | bunt
+
+- Die höchste Karte wird negiert
```

Comment:
```suggestion
- Die höchste Karte in dem Stich wird negiert/zählt nicht.
```

Reply:
Die Fee wird gekillt. Der Drache gewinnt.

Reply:
Dann schreibe diesen Sonderfall bitte mit in die Beschreibung!